prombench/manifests/cert-manager/1c_issuer.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-cluster-issuer
+spec:
+  acme:
+    server: https://acme-v02.api.letsencrypt.org/directory
+    email: vanditsinghkv@gmail.com
```

Comment:
Maybe an email Prometheus specific?

Reply:
Whose email should I put here? I added mine when testing locally.